Tests/LibWeb/Text/input/wpt-import/css/css-values/urls/#foo

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
```

Comment:
Not sure where this came from but I think it's safe to say it's not necessary.

Reply:
Oh whoops! I just did `Meta/WPT.sh import css/css-values/urls/`. My guess is the `fragment-only.html` test below has some `url(#foo)`s in it and we're not smart enough to ignore those. :sweat_smile: